addons/website/static/src/js/content/menu.js

```diff
@@ -603,6 +603,19 @@ publicWidget.registry.hoverableDropdown = animations.Animation.extend({
             this.$dropdownMenus.css('top', '');
         }
     },
+    /**
+     * @private
+     */
+    _toggleDropdown: function (ev) {
+        let { currentTarget } = ev;
```

Comment:
I think I would rater read `ev.currentTarget` at the two places you need it but if you prefer this, it's fine, but use `const` instead of `let` :)